### PR TITLE
describe processing for CMs with presentation definitions

### PR DESCRIPTION
**Pull request by kimdhamilton**
Attempts to address #125.

Follow up action: create an issue on PE to include more information about packaging of the response (per `format`) property.

Also to discuss on Thursday's call: branch protection 😬

### CODE REVIEW

**Review by decentralgabe — APPROVED** [review]

File: spec/spec.md [hunk 1]
```diff
@@ -284,6 +284,10 @@ DIDComms   | `$.presentations~attach.data.json`
 VP         | top-level
 CHAPI      | `$.data`
 
+### Input Evaluation
+
+A conformant consumer of a [[ref:Credential Manifest]] containing a `presentation_definition` property ****MUST**** include a valid [Presentation Submission](https://identity.foundation/presentation-exchange/#presentation-submission) in the  `presentation_submission` property of its corresponding [[ref:Credential Application]]. The [Presentation Submission](https://identity.foundation/presentation-exchange/#presentation-submission) must be constructed as specified in [Presentation Exchange: Input Evaluation](https://identity.foundation/presentation-exchange/#input-evaluation).
```
Comment (decentralgabe) [inline spec/spec.md:289]:
```suggestion
A conformant consumer of a [[ref:Credential Manifest]] containing a `presentation_definition` property ****MUST**** include a valid [Presentation Submission](https://identity.foundation/presentation-exchange/#presentation-submission) in the  `presentation_submission` property of its corresponding [[ref:Credential Application]]. The [Presentation Submission](https://identity.foundation/presentation-exchange/#presentation-submission) must be processed as specified in [Presentation Exchange: Input Evaluation](https://identity.foundation/presentation-exchange/#input-evaluation).
```

Comment (kimdhamilton) [reply spec/spec.md:289]: Note to self: is conformant consumer the right term?

Comment (decentralgabe) [reply spec/spec.md:289]: `processor`?

Comment (decentralgabe) [reply spec/spec.md:289]: `applicant`?

Comment (decentralgabe) [reply spec/spec.md:289]: one other comment - there are two types of processing
1. the applicant needs to process the manifest's submission and generate a compliant submission as a part of their application
2. the creator of the manifest (issuer?) needs to then process the credential application's presentation submission according to the pe spec

File: spec/spec.md [hunk 2]
```diff
@@ -284,6 +284,10 @@ DIDComms   | `$.presentations~attach.data.json`
 VP         | top-level
 CHAPI      | `$.data`
 
+### Input Evaluation
+
+If a [[ref:Credential Manifest]] includes a `presentation_definition` property, the [[ref:User Agent]] ****MUST**** include a valid [Presentation Submission](https://identity.foundation/presentation-exchange/#presentation-submission) in the  `presentation_submission` property of its corresponding [[ref:Credential Application]]. The [Presentation Submission](https://identity.foundation/presentation-exchange/#presentation-submission) must be processed as specified in [Presentation Exchange: Input Evaluation](https://identity.foundation/presentation-exchange/#input-evaluation).
```
Comment (brentzundel) [inline spec/spec.md:289]:
```suggestion
If a [[ref:Credential Manifest]] includes a `presentation_definition` property, the [[ref:User Agent]] ****MUST**** include a valid [Presentation Submission](https://identity.foundation/presentation-exchange/#presentation-submission) in the  `presentation_submission` property of its corresponding [[ref:Credential Application]]. The [Presentation Submission](https://identity.foundation/presentation-exchange/#presentation-submission) ****MUST**** be processed as specified in [Presentation Exchange: Input Evaluation](https://identity.foundation/presentation-exchange/#input-evaluation).
```

**Pull request merged**